robo.yml

```diff
@@ -40,8 +40,9 @@ data:
 
   - name: "europa-digital-thesaurus"
     graph: "http://data.europa.eu/uxp"
-    url: "https://raw.githubusercontent.com/openeuropa/triple-store-dev/master/local_config/digital-europa-thesaurus.rdf"
-    format: "rdf"
+    url: "http://publications.europa.eu/resource/cellar/1deb8f26-d93d-11e9-9c4e-01aa75ed71a1.0001.01/DOC_1"
+    file: "skos-core.rdf"
+    format: "zip"
```

Comment:
When building I get a 404:

![image](https://user-images.githubusercontent.com/153362/65862632-f3574000-e36e-11e9-83bf-02533fbf9d78.png)

And indeed that URL seems not to point to any resource.

Reply:
The URL I found on the [DET page](https://publications.europa.eu/en/web/eu-vocabularies/at-dataset/-/resource/dataset/det) seems to be this one:

```
https://publications.europa.eu/euvoc-portlet/download?cellarURI=http%3A%2F%2Fpublications.europa.eu%2Fresource%2Fcellar%2F1deb8f26-d93d-11e9-9c4e-01aa75ed71a1.0001.02%2FDOC_1&fileName=skos-core.rdf
```

It also seems not to be a ZIP file.